packages/bsky/src/context.ts

```diff
@@ -91,6 +98,23 @@ export class AppContext {
     return auth.roleVerifier(this.cfg)
   }
 
+  async serviceAuthJwt(aud: string) {
+    const iss = this.cfg.serverDid
+    return createServiceJwt({
+      iss,
+      aud,
+      keypair: this.signingKey,
+    })
+  }
+
+  async pdsAdminAgent(did: string): Promise<AtpAgent> {
+    const data = await this.idResolver.did.resolveAtprotoData(did)
+    const agent = new AtpAgent({ service: data.pds })
+    const jwt = await this.serviceAuthJwt(did)
```

Comment:
unsure of having the audience be the did in question, but seemed like the easiest thing for now 🤔 

we'll need to be careful to not have the appview sign tokens for other purposes in the meantime

Reply:
Hmmm!  Bit of a thinker 🤔

Reply:
kk i left this with the did of the user as the `aud` & I explicitly verify the `aud` matches the request in question on the PDS (which dramatically reduces blast-radius of one of these leaking)